### DISCUSSION TITLE
Alpine breaks jQuery in IE

### DISCUSSION THREAD

**Discussion by skttl** [alpinejs/alpine#525] [4. General]
Old legacy site here, using jquery and jquery.validate (because of Microsofts jQuery Validate Unobtrusive).

When loading pages in IE containing both jQuery and Alpine, I get this exception:
![image](https://user-images.githubusercontent.com/3726467/82807017-41353100-9e87-11ea-80f7-c698c46e3070.png)

Apparently jQuery doesn't like modificiations to the Object prototype. If I remove Alpine, it works as expected (except the Alpine parts)

Anyone know how to work around this? Removing jQuery is not an option, even though it would be the best :)

**Comment by HugoDF** [comment]
Do you have some sample HTML for this?
